recipes/powershell.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Cookbook Name:: windows-hardening
+# Recipe:: powershell
+#
+# Copyright (c) 2016 Joe Gardiner, All Rights Reserved.
+
+# Enabling PowerShell script block logging will record detailed information from the processing of PowerShell commands and scripts
+# Powershell ScriptBlock Logging
+registry_key 'HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\PowerShell\ScriptBlockLogging' do
```

Comment:
Can you reference where those requirements are coming from? We need to update our baseline then too https://github.com/dev-sec/windows-baseline